script/parse-examples

```diff
@@ -13,6 +13,7 @@ function checkout() {
 
   pushd "$repo"
   git fetch && git reset --hard "$sha"
+  rm -rf .git
```

Comment:
Right now, the script expects the git directory to be kept. If the folder already exists, but the sha has been bumped, it will try to run `git fetch && git reset --hard`.

Was CI failing because of a file in the `.git` dir? Instead, maybe you could add that to the `known_failures.txt` file.

Reply:
> Was CI failing because of a file in the .git dir?

Locally yeah, but I'm not getting this anymore, so all good. I have reverted that change.